packages/storybook-blocks/jest.config.ts

```diff
@@ -14,7 +14,7 @@ module.exports = {
   preset: 'ts-jest/presets/default',
   testEnvironment: 'jest-environment-jsdom',
   testMatch: ['**/__tests__/**/*.test.[jt]s?(x)'],
-  transformIgnorePatterns: ['/node_modules/(?!monday-ui-react-core)'],
+  transformIgnorePatterns: ['/node_modules/(?!@vibe/core)'],
```

Comment:
it sound like a mistake, why is it inside storybook-blocks?

Reply:
You're right, removing